### PR TITLE
feat: update candy machine list

### LINKED ISSUE #91: [Task]: Update candy machines list UI to new design
**Task description.**

Update candy machine list UI. 
Please follow the design in figma https://www.figma.com/file/Fe2hyux4KGmKjeni2nb2mg/Sugar-Rush?node-id=12%3A4964
image

**Requirements.**

- Create new `CandyMachineTile` component 
image

### PR DESCRIPTION
**Pull request by evavirseda**
Fixes #91 

## Changes

  - Update candy machine list UI.
  - Create new `CandyMachineTile` component
   
### Checks

* [x] Have you catchup your branch with the latest state in base?
* [x] Have you lint your code locally prior to submission?
* [x] Have you reviewed your proposed changes and removed debris?
* [x] Have you successfully ran due tests with your changes?

### CODE REVIEW

File: components/CandyMachineList.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { CandyMachineTile } from 'components'
+import { FC, useState } from 'react'
+import { useRecoilValue } from 'recoil'
+const CandyMachineList: FC<{
+    candyMachineAccounts: string[]
+    setCandyMachineAccounts: React.Dispatch<React.SetStateAction<string[]>>
+}> = ({ candyMachineAccounts, setCandyMachineAccounts }) => {
+    const searchInput = useRecoilValue(candyMachineSearchState)
+    const [isOpen, setIsOpen] = useState(false)
+    const [selectedAccount, setSelectedAccount] = useState('')
```
Comment (cpl121) [inline components/CandyMachineList.tsx:13]: Not used `setSelectedAccount` so `selectedAccount` is always ''

**Review by cpl121 — APPROVED** [review]

File: pages/index.tsx [hunk 2]
```diff
@@ -55,14 +56,21 @@ const ManageCandyMachines: NextPage = () => {
...
-
-                {!error && accounts.length === 0 && <Spinner />}
+            <div className='flex-col'>
+                <div className='d-flex my-5 flex-items-center'>
+                    <h2>Candy Machines ·</h2>
+                    <h2 className='ml-2'>
+                        {!error && accounts.length === 0 ? (
+                            <Spinner size='small' sx={{ display: 'flex' }} />
+                        ) : (
+                            accounts?.length
```
Comment (VmMad) [inline pages/index.tsx:70]: This shows 1 even when you don't have any CM's because default state is [' ']

**Review by VmMad — CHANGES_REQUESTED** [review]
Just one small change!

**Review by VmMad — APPROVED** [review]
:medal_sports:

**Pull request merged**